tests/parallelDeployAcceptance/features/bootstrap/bootstrap.php

```diff
@@ -25,9 +25,9 @@
 $classLoader = new ClassLoader();
 $classLoader->addPsr4(
 	"",
-	__DIR__ . "/../../../tests/acceptance/features/bootstrap",
+	__DIR__ . "/../../../tests/acceptance/bootstrap",
 	true
 );
 
-$classLoader->addPsr4("TestHelpers\\", __DIR__ . "/../../../TestHelpers", true);
+$classLoader->addPsr4("TestHelpers\\", __DIR__ . "/../../../acceptance/TestHelpers", true);
```

Comment:
you can leave this file as it is. no changes required.

Reply:
Okay. Makes sense as we are deleting those anyway.